packages/message/src/util/hash.ts

```diff
@@ -0,0 +1,5 @@
+import sha256 from 'crypto-js/sha256';
+
+export function hash(msg: string): string {
+  return sha256(msg).toString();
+}
```

Comment:
Please use the `utils` folder name.
And in each folder, export all functions from ʻindex.ts` in same directory (`utils/index.ts`).